### PR TITLE
Swap out 3rd party font for system fonts

### PR DESCRIPTION
**Pull request by mixonic**
Ember CLI Addon Docs should use system fonts by default. System fonts load quickly for users, reduce the site's coupling to 3rd party assets, and make sites privacy-friendly by default.

The Google font previously relied upon would result in Google having the ability to add cookies for that traffic, possibly placing a site out of compliance with EU and other regulations regarding user tracking without consent processes in place.

Here I've replaced (via https://github.com/Addepar/ember-table/pull/983 for the purposes of illustration):

* https://fonts.google.com/specimen/Crimson+Text
* Try it in your browser at https://opensource.addepar.com/ember-table/versions/v5.0.3/docs

Screenshot of previous font on macOS
Screenshot of previous font on Ubuntu

with

* https://github.com/system-fonts/modern-font-stacks#old-style
* Try it in your browser at https://opensource.addepar.com/ember-table/versions/master/docs

Screenshot of new font on macOS
Screenshot of new font on Ubuntu

Note that the strategy of font stacks is to pick a class of font, so this will render differently on Linux vs. macOS, etc.

I'm not sure what kind of release this warrants, but I'm going to suggest **minor**. It is not a breaking change, though it is a visual change.

Fixes https://github.com/ember-learn/ember-cli-addon-docs/issues/1415

### CODE REVIEW

**Comment by mixonic** [comment]
I'm taking the initiative to land this based on the consensus that any solution was acceptable expressed at https://github.com/ember-learn/ember-cli-addon-docs/issues/1415.

Ping me if there is anything to follow up on! I'll look into triggering a release in the next day or two.

File: addon/styles/tailwind.config.js
Comment (c69-addepar) [inline addon/styles/tailwind.config.js:148]: With `serif` fallback, it should work pretty much anywhere, but still - maybe its worth adding a windows font to the fallback list ?

**Review by c69-addepar — APPROVED** [review]
LGTM but small comment

File: addon/styles/tailwind.config.js
Comment (mixonic) [reply addon/styles/tailwind.config.js:148]: Per modern font stacks, `Palatino Linotype` is available on Windows XP and up.

**Comment by mixonic** [comment]
@RobbieTheWagner I haven't been able to make a release cut yet, so please feel free to provide feedback and I will address it!

I'm also blocked on making a release until I'm granted permissions on the npm package, if you can unblock that. (@dfreeman was going to help me out)

**Pull request merged**
